src/sass/rx/partials/_rx-prescriptions.scss

```diff
@@ -50,7 +50,7 @@
 
   a {
     text-decoration: none;
-    
+
```

Comment:
We have a bunch of trailing spaces among the stylesheets. Might be nice to add a check for that to the linter.